pom.xml

```diff
@@ -17,7 +17,7 @@
         <delombok.source>${project.build.directory}/delombok-sources</delombok.source>
 
         <lombok.version>1.18.20</lombok.version>
-        <gson.version>2.8.6</gson.version>
+        <jackson.version>2.12.5</jackson.version>
```

Comment:
GSON already connected at 92 line. 
Jackson doesnt defined, define it like:
```
<dependencyManagement>
    ...
    <dependency>
        <groupId>com.fasterxml.jackson</groupId>
        <artifactId>jackson-bom</artifactId>
        <version>${jackson.version}</version>
        <type>pom</type>
        <scope>import</scope>
    </dependency>
</dependencyManagement>

<dependencies>
    ...
    <dependency>
        <groupId>com.fasterxml.jackson.core</groupId>
        <artifactId>jackson-core</artifactId>
    </dependency>
    <dependency>
        <groupId>com.fasterxml.jackson.core</groupId>
        <artifactId>jackson-databind</artifactId>
    </dependency>
</dependencies>
```

Reply:
Now fixed